### PR TITLE
[analyze] Removing long enabled checker list at info log level

### PR DESCRIPTION
**Pull request by dkrupp**
At every analysis at the defult log level there was a wall of text checker list. It is enough to print this list at the verbose log level and redirect the user to the metadat.json file at default log level.

### CODE REVIEW

File: analyzer/codechecker_analyzer/analyzer.py [hunk 1]
```diff
@@ -231,8 +231,9 @@ def perform_analysis(args, skip_handlers, actions, metadata_tool,
         metadata_info['analyzer_statistics']['version'] = version
 
         metadata_tool['analyzers'][analyzer] = metadata_info
-
-    LOG.info("Enabled checkers:\n%s", '\n'.join(
+    LOG.info("Enabled checker list can be found in " +
+             os.path.join(args.output_path, "metadata.json"))
```
Comment (whisperity) [inline analyzer/codechecker_analyzer/analyzer.py:235]:
```suggestion
    LOG.info("Enabled checker list can be found in %s",
             os.path.join(args.output_path, "metadata.json"))
```

**Comment by Szelethus** [comment]
I totally support this patch -- the default output is still let a little loose on verbosity. With that said, metadata.json isn't very human readable without formatting, so we should consider prettifying it.

**Comment by whisperity** [comment]
@Szelethus The `metadata.json` is not meant to be human-readable. It is a completely internal format and transitional medium, we should not suggest, direct, or expect users to do anything with it.

`CodeChecker parse` `--details`(?), however, could emit this list. Maybe as a proper list, not a huge concatenated string, which was also barely readable due to the difference in lengths. `clang-tidy --list-checks --checks='*'` emits a formatted list, one element per line.

**Review by whisperity — CHANGES_REQUESTED** [review]
&nbsp;

File: analyzer/codechecker_analyzer/analyzer.py (hunk 1 above)
Comment (whisperity) [inline analyzer/codechecker_analyzer/analyzer.py:235]:
```suggestion
```

File: analyzer/codechecker_analyzer/analyzer.py [hunk 2]
```diff
@@ -231,8 +231,9 @@ def perform_analysis(args, skip_handlers, actions, metadata_tool,
...
+    LOG.debug("Enabled checkers:\n%s", '\n'.join(
         k + ': ' + ', '.join(v) for k, v in enabled_checkers.items()))
```
Comment (whisperity) [inline analyzer/codechecker_analyzer/analyzer.py:237]: Good, but maybe could be formatted into one line for each checker.

**Review by whisperity — CHANGES_REQUESTED** [review]
(Reasoning above.)

**Review by whisperity — APPROVED** [review]

**Pull request merged**